setup.py

```diff
@@ -34,7 +34,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.1.4',
+    version='2.1.5',
```

Comment:
@noraiz-anwar Have you created the release tag?
https://github.com/edx/edx-ora2/releases/new

Reply:
Not yet. Is this supposed to be created before merging this version update?

Reply:
@awaisdar001 I have communicated with @efischer19 to confirm that this PR needs to be merged before so that newly created tag can point to this commit.